hack/test-e2e-node.sh

```diff
@@ -35,6 +35,19 @@ if [[ -z "${GOPATH}" ]]; then
   exit 1
 fi
 
+# Update ip firewall
+# We need to add rules to accept all TCP/UDP/ICMP packets.
+if sudo iptables -L INPUT | grep "Chain INPUT (policy DROP)" > /dev/null; then
+	sudo iptables -A INPUT -w -p TCP -j ACCEPT
```

Comment:
Would this work better:
```sudo iptables -I INPUT 1 -w -j ACCEPT```
^ translates to insert at rule 1 in ACCEPT allow traffic for `all` protocols from anywhere to anywhere

Reply:
Discussed with our networking team. I believe we want to keep the existing filter, and accept at the end.